### PR TITLE
cmd/swarm: add feature to specify account password non-interactively

### PR DESCRIPTION
**Pull request by farazdagi**
- `swarm` command accepts `--bzzacount` flag, but password must be provided interactively by typing
- `geth` command has the `--password` flag (where one can provide password file to be used to unlock account in a non-interactive mode)
- this PR adds `--password` flag to `swarm` command allowing users to start swarm node in completely non-interactive manner

**Note:** For consistency, code mimics the same in `cmd/geth` (it seems it already has been derived from there anyway).

### CODE REVIEW

File: cmd/swarm/main.go [hunk 1]
```diff
@@ -228,6 +228,7 @@ Cleans database of corrupted entries.
 		SwarmSyncEnabledFlag,
 		SwarmPortFlag,
 		SwarmAccountFlag,
+		utils.PasswordFileFlag,
```
Comment (zelig) [inline cmd/swarm/main.go]: please add this among the middle ones

**Comment by farazdagi** [comment]
@zelig Done!

**Comment by zelig** [comment]
@farazdagi could you please rebase and finalise this for merge? sorry for delay, waited for #3742

**Comment by farazdagi** [comment]
@zelig done!

**Pull request merged**